lib/3scale_toolbox/attribute_filters/service_id_from_ref_filter.rb

```diff
@@ -0,0 +1,30 @@
+module ThreeScaleToolbox
+  module AttributeFilters
+    class ServiceIDFilterFromServiceRef
```

Comment:
What I would expect from class called `ServiceIDFilterFromServiceRef` is being initialized with some service_id value, instead of getting `remote` and service_ref.  The class also should know field name, service_id_key does not make sense IMO.

If service_id is nil, the filter should reject all values.

Reply:
> The class also should know field name, service_id_key does not make sense IMO.

The idea behind making the service_id_key configurable is that depending on the context the key name might change. For example, if using in a context where service is central (the service command for example) it's returned as "id" by System instead of by "service_id" :thinking:

Reply:
> What I would expect from class called ServiceIDFilterFromServiceRef is being initialized with some service_id value, instead of getting remote and service_ref.

The idea behind this was to hide the logic of finding the service in the class itself so it would have not to be duplicated in every command code (client code of this class). Don't you see this approach suitable? You enter a service ref and you get the specified service refs from the list filtered

Reply:
Ok, let's go for it